src/test/java/org/beanpod/switchboard/fixture/DecoderFixture.java

```diff
@@ -15,57 +13,51 @@
 
 public class DecoderFixture {
     public static final String SERIAL_NUMBER = "1";
+    public static final String SERIAL_NUMBER2 = "2";
     private static String pattern = "yyyy-MM-dd HH:mm:ss";
     private static SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
     static private Set<InputChannelEntity> setOfChannels = ChannelFixture.getInputChannelEntities();
 
-    public static DecoderEntity getDecoder1(DeviceEntity device){
-        try {
-            return new DecoderEntity("1",
-                    simpleDateFormat.parse("2020-10-31 05:05:05"),
-                    device,
-                    setOfChannels);
-        } catch (ParseException e) {
-            e.printStackTrace();
-            throw new RuntimeException("Unable to parse the date in Decoder 1");
-        }
+    public static DecoderEntity getDecoderEntity1() throws ParseException {
+        return DecoderEntity.builder()
+                .serialNumber(SERIAL_NUMBER)
+                .lastCommunication(simpleDateFormat.parse("2020-10-31 05:05:05"))
+                .device(DeviceFixture.getDevice1())
+                .input(null)
+                .build();
     }
 
-    public static DecoderEntity getDecoder2(DeviceEntity device){
-        try {
-            return new DecoderEntity("1",
-                    simpleDateFormat.parse("2020-10-31 05:05:05"),
-                    device,
-                    setOfChannels);
-        } catch (ParseException e) {
-            e.printStackTrace();
-            throw new RuntimeException("Unable to parse the date in Decoder 1");
-        }
+    public static DecoderEntity getDecoderEntity2() throws ParseException {
+        return DecoderEntity.builder()
+                .serialNumber(SERIAL_NUMBER2)
+                .lastCommunication(simpleDateFormat.parse("2020-10-31 05:05:05"))
+                .device(DeviceFixture.getDevice1())
+                .input(null)
+                .build();
```

Comment:
Could we use `@SneakyThrows` on these instead?

Reply:
YES SIR, fixed